test/api/auth_test.rb

```diff
@@ -0,0 +1,36 @@
+require 'test_helper'
+
+class AuthTest < MiniTest::Test
+  include Rack::Test::Methods
+
+  def app
+    Rails.application
+  end
+
+  def test_auth_post
+    # Get response back for logging in with username 'acain' password 'password'
+    post '/api/auth.json', '{"username":"acain", "password":"password"}', "CONTENT_TYPE" => 'application/json'
```

Comment:
@Reubsinit Awesome! Where did you end up finding how to do this? Or was it literally just playing with the library code?